### PR TITLE
AMA | Evaluate Decision | Show the date that the case was assigned to an attorney

### PR DESCRIPTION
**Pull request by evankroske**
Resolves https://github.com/department-of-veterans-affairs/caseflow/issues/6964

### Description
Shows the date that the judge asked the attorney to write a decision on the Evaluate Decision page for AMA cases.

### Testing Plan
- [x] `rake db:setup db:seed`
- [x] Become Bvaaabshire.
- [x] /queue
- [x] Open an AMA case.
- [x] Select Ready for Dispatch.
- [x] Click Continue.
- [x] Set both dispositions to Allowed.
- [x] Click Continue.
- [x] "Date assigned to attorney" should be the date at which you ran the rake command.

### CODE REVIEW

File: spec/factories/task.rb [hunk 1]
```diff
@@ -53,7 +53,7 @@
       appeal { create(:appeal) }
     end
 
-    factory :ama_judge_task do
+    factory :ama_judge_task, class: JudgeTask do
       type "JudgeTask"
```
Comment (aroltsch) [inline spec/factories/task.rb]: Oh cool! Can we remove the `type` from line 57 then?

File: app/models/tasks/judge_task.rb [hunk 2]
```diff
@@ -9,4 +9,9 @@ def when_child_task_completed
     update!(action: :review)
     super
   end
+
+  def previous_task
+    fail Caseflow::Error::TooManyChildTasks, task_id: id if children.length > 1
+    children[0]
```
Comment (aroltsch) [inline app/models/tasks/judge_task.rb]: A judge task might have another type of child. I think we talked about having a judge to assign directly to the hearing schedule team or similar. We should filter by the children types here. 
```children.where(type: "AttorneyTask")```

File: spec/factories/task.rb (hunk 1 above)
Comment (evankroske) [reply spec/factories/task.rb]: No, I don't think that FactoryBot understands STI. I think that we need type and class

File: app/models/tasks/judge_task.rb (hunk 2 above)
Comment (evankroske) [reply app/models/tasks/judge_task.rb]: OK, I just made this change

**Pull request merged**